### PR TITLE
Add docstring for load_checkpoint

### PR DESCRIPTION
**Pull request by eddogola**
## Description

Added docstrings to load_checkpoint module and _get_file_to_weight_map function.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by eddogola** [comment]
@kwen2501 @wz337

File: pippy/LoadModule.py [hunk 1]
```diff
@@ -20,7 +20,22 @@ def load_checkpoint(
     device: torch.device = None,
     dtype: torch.dtype = None,
     checkpoint_prefix: str = None,
-):
+) -> nn.Module:
+    """
+    Load a checkpoint from a model file.
+    Args:
+        model: the model to load the checkpoint into
```
Comment (wz337) [inline pippy/LoadModule.py:27]: nit: for consistency, let's also include the variable type in args as you did for _get_file_to_weight_map()?

File: pippy/LoadModule.py [hunk 2]
```diff
@@ -216,7 +242,7 @@ def _set_module_tensor_to_device(
             # Note: `torch.tensor()` allocates new memory to copy the data of
             # tensor, so clone is taken care of
             new_value = torch.tensor(value, device=device, dtype=dtype)
-
+
```
Comment (kwen2501) [inline pippy/LoadModule.py:245]: Unintentional whitespace?

**Review by kwen2501 — APPROVED** [review]
LGTM!
Congrats on the first PR!

**Review by kwen2501 — APPROVED** [review]
LGTM!
Congrats on the first PR!

**Review by kwen2501 — APPROVED** [review]
LGTM!
Congrats on the first PR!

**Pull request merged**